### DISCUSSION TITLE
Toolbar for thumb carousel

### DISCUSSION THREAD

**Discussion by Patrik17671** [fancyapps/ui#292] [Q&A]
Hey , it is possible to allow arrows for thumb carousel ? My carousel looks like [this](https://fancyapps.com/playground/vu) in options i set "Navigation: true" but nothing happened still have arrows just for main carousel .
Thx for answer.

**Comment by fancyapps** [comment]
Hi,

Why don't you share URL to your attempt? It should work fine - https://fancyapps.com/playground/2Q8
